src/runtime/web/index.js

```diff
@@ -40,9 +40,14 @@ function init() {
   this.views.push(
     document.l10n = new View(this, document));
 
+  var prevLangs = [];
+  if (document.documentElement.hasAttribute('lang')) {
```

Comment:
Also, `prevLangs` is really the previously negotiated user's lang chain. I'm not sure it's the right abstraction for storing the language the `document` was pretranslated into.